tensorbay/opendataset/UAVDT/loader.py

```diff
@@ -0,0 +1,175 @@
+                sequence_attributes_path, dataset.catalog.classification.attributes.keys()
+            )
+            classification = Classification(attributes=sequence_attributes)
+
+            frame_id_ground_truth_map = _extract_gt_txt_to_dict(
+                root_path,
+                sequence,
+                dataset.catalog.box2d,
+            )
+
+            image_paths = glob(os.path.join(root_path, "UAV-benchmark-M", sequence, "*.jpg"))
+            for image_path in image_paths:
+                data = Data(image_path)
+                # The image_name looks like `img<frame_id>.jpg`.
+                # The frame_id is consist of six digital numbers.
+                ground_truths = frame_id_ground_truth_map[int(os.path.basename(image_path)[3:9])]
+                data.label.classification = classification
+                data.label.box2d = _convert_ground_truths_to_labels(ground_truths)
+                segment.append(data)
+
+    return dataset
+
+
+def _extract_attr_txt_to_dict(
+    sequence_attributes_path: str, attribute_names: Tuple[str, ...]
+) -> Dict[str, bool]:
+    with open(sequence_attributes_path) as fp:
+        csv_reader = csv.reader(fp)
+        elements = next(csv_reader)
+        attributes = {
+            attribute_name: bool(int(value))
+            for attribute_name, value in zip(attribute_names, elements)
+        }
+
+    return attributes
+
+
+def _extract_gt_txt_to_dict(
+    path: str,
+    sequence: str,
+    box2d_catalog: Box2DSubcatalog,
+) -> Dict[int, List[Dict[str, Any]]]:
+    attributes = box2d_catalog.attributes
+    catagories = box2d_catalog.categories.keys()
+
+    ground_truth_path = os.path.join(path, "UAV-benchmark-MOTD_v1.0", "GT")
+    frame_id_ground_truth_map = defaultdict(list)
+    with open(os.path.join(ground_truth_path, f"{sequence}_gt.txt")) as file_mot, open(
+        os.path.join(ground_truth_path, f"{sequence}_gt_whole.txt")
+    ) as file_det:
+        mot_csv = csv.DictReader(
+            file_mot,
+            fieldnames=[
+                "frame_index",
+                "target_id",
+                "bbox_left",
+                "bbox_top",
+                "bbox_width",
+                "bbox_height",
+                "score",
+                "in_view",
+                "occlusion",
+            ],
+        )
+        det_csv = csv.DictReader(
+            file_det,
+            fieldnames=[
+                "frame_index",
+                "target_id",
+                "bbox_left",
+                "bbox_top",
+                "bbox_width",
+                "bbox_height",
+                "out_of_view",
+                "occlusion",
+                "boject_category",
+            ],
+        )
+        for elements_mot, elements_det in zip(mot_csv, det_csv):
+            ground_truth: Dict[str, Any] = {}
+            ground_truth["target_id"] = elements_det["target_id"]
+            ground_truth["bbox_xywh"] = [
```

Comment:
can be construct `LabeledBox2D` here, rather than use another function to convert dict to `LabeledBoxed2D`